bstrp_functions.sh

```diff
@@ -457,3 +475,5 @@ read_feature_options(){
     *) echo -e "${RED}Please enter an option A-Z or AA-AF or 1-7...${NO_COLOR}" && sleep 2
   esac
 }
+
+# vim: shiftwidth=2 tabstop=2 expandtab
```

Comment:
did you add these vim-specific comments intentionally?

Reply:
yes